azure-toolkit-libs/pom.xml

```diff
@@ -54,62 +54,67 @@
 
         <azure.toolkit-lib.version>${project.version}</azure.toolkit-lib.version>
 
-        <azure.core.version>1.15.0</azure.core.version>
-        <azure.aad.version>1.6.4</azure.aad.version>
+        <azure.core.version>1.16.0</azure.core.version>
+        <azure.aad.version>1.6.7</azure.aad.version>
         <azure.applicationinsights.version>2.6.3</azure.applicationinsights.version>
         <azure.applicationinsights-web.version>[2.0,)</azure.applicationinsights-web.version>
         <azure.azure-mgmt-appplatform.version>1.0.0-beta</azure.azure-mgmt-appplatform.version>
         <azure.client.version>1.7.12</azure.client.version>
-        <azure.identity.version>1.3.0-beta.2</azure.identity.version>
-        <azure.resourcemanager.version>2.4.0</azure.resourcemanager.version>
-        <azure.storage.version>8.0.0</azure.storage.version>
-        <azure.version>1.38.1</azure.version>
-        <azure.core.http.okhttp.version>1.6.1</azure.core.http.okhttp.version>
-        <azure.storage-file-share.version>12.9.0</azure.storage-file-share.version>
+        <azure.identity.version>1.3.0</azure.identity.version>
+        <azure.resourcemanager.version>2.5.0</azure.resourcemanager.version>
+        <azure.storage.version>8.6.6</azure.storage.version>
+        <azure.version>1.41.1</azure.version>
+        <azure.eventhubs.version>3.2.2</azure.eventhubs.version>
+        <azure.eventhubs-eph.version>3.2.2</azure.eventhubs-eph.version>
```

Comment:
can we merge `azure.eventhubs-eph.version` and `azure.eventhubs.version`?

Reply:
merged.